hiv-benchmarking/requirements.txt

```diff
@@ -0,0 +1,8 @@
+Bio==1.7.1
+biopython==1.84
+matplotlib==3.8.0
+numpy==1.24.4
+pandas==1.5.3
+Requests==2.32.3
+scikit_learn==1.2.2
```

Comment:
I get this error when trying to build conda env with requirement.txt file and command `conda env create -f hiv-benchmarking/requirements.txt --name hiv_benchmarking`

Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound: 
  - scikit_learn==1.2.2

Reply:
I also get the same error with `conda install --file requirements.txt`

Reply:
@ntalluri once it was changed to `scikit-learn` did pip installing the requirements file work? just wanted to double check

Reply:
I was able to install it with
```
$ conda create --name hiv_benchmarking python=3.10
$ conda activate hiv_benchmarking
$ pip install -r requirements.txt
```